code/modules/mod/modules/modules_maint.dm

```diff
@@ -48,6 +48,9 @@
 	mod.wearer.emote("scream")
 	playsound(mod.wearer, 'sound/effects/snap.ogg', 75, TRUE, frequency = 0.5)
 	playsound(mod.wearer, 'sound/effects/splat.ogg', 50, TRUE, frequency = 0.5)
+
+	mod.wearer.client?.give_award(/datum/award/achievement/misc/springlock, mod.wearer)
+
```

Comment:
no need for the empty spaces

Reply:
I like it because it’s doing something else than the other modsuit code, but it’s your feature so I respect that